assets/src/components/routePicker.tsx

```diff
@@ -50,37 +68,76 @@ const SelectedRoutesList = ({
   )
 }
 
+const RouteFilterInterface = ({
```

Comment:
What do you think about having a `routeFilter.tsx` that exports the `<RouteFilterInterface>` and the `useRouteFilter` hook?

Reply:
I had the same thought too, and was initially attempting to do that. The thing I was getting hung up on is that`RouteFilterInterface` requires the state values that are returned by `useRouteFilter`. If you can think of something I'm missing to make that work I'd be happy to hear it.

Reply:
The `RouteFilter` type would also be exported from `routeFilter.ts`, and `RoutePicker` would pass them in. Exact same setup as now, just moving the code into a different file.

Reply:
Oh I see, yep that works, I just had a blind spot trying to hide the filter data away. Thanks!